main.py

```
@@ -1,20 +1,310 @@
+                return
+
+    elif q.data == "paid_list":
+        kb = [[InlineKeyboardButton(f"📄 {f['name']} - ${f['price']}", callback_data=f"buy_{f['id']}")] for f in data if f["type"] == "paid"]
+        await q.message.reply_text("💰 Paid Files:", reply_markup=InlineKeyboardMarkup(kb) if kb else back_button(is_admin(uid)))
+
+    elif q.data.startswith("buy_"):
+        fid = q.data.replace("buy_", "")
+        for f in data:
+            if f["id"] == fid:
+                await context.bot.send_message(OWNER_ID, f"📥 Paid file request from user {uid}:\nFile: {f['name']}\nPrice: ${f['price']}")
+                await q.message.reply_text(f"✅ Purchase request sent to admin.")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)